tools/run_tests/run_tests.py

```diff
@@ -759,8 +759,12 @@ def dockerfile_dir(self):
             self.python_manager_name(), _docker_arch_suffix(self.args.arch))
 
     def python_manager_name(self):
-        if self.args.compiler in ['python3.5', 'python3.6']:
-            return 'pyenv'
+        if self.args.compiler == 'python3.5':
```

Comment:
If we name the directories appropriately, perhaps we could get away with `return 'stretch_' + self.args.compiler` instead of the switch-case boilerplate.

Reply:
In this case, the Dockerfile name will become `python_stretch_python3.5_x64/Dockerfile`. The `python` appeared twice, is that okay?

Reply:
I prefer to only take the number out. I did it in `if-else` clause to make it more clear, and it can be done by one line of string manipulation.